templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/public_index.html.twig

```diff
@@ -4,6 +4,7 @@
 
 {% block component_part %}
     {% apply spaceless -%}
+        <h1 class="hide-visually">{{ "public.index.heading"|trans }}</h1>
```

Comment:
While this is our "hand-rolled" class for visually hiding stuff, the [Tailwind class](https://tailwindcss.com/docs/screen-readers#screen-reader-only-elements) (which can already be used) is `sr-only`. 

However as we're using `hide-visually` all over the place, we could as well refacor it all at once.

Regarding the suggestion below, the `trans` call would change to:

```suggestion
        <h1 class="hide-visually">{{ "public.index.heading"|trans({ projectName: projectName } }}</h1>
```